docs/cheat-sheet.rst

```diff
@@ -230,6 +229,7 @@ If the information in the *designs* dataset completely defines the technology an
   
 Including the Offset value in the *parameters* dataset creates a user reference that makes it easier to access parameter values when defining the technology model.
 
+**Mandatory data.** The *parameters* dataset is required to exist and to include at least one Parameter for every Technology-Scenario combination. If there are no Parameters present in the technology model, then the Parameter may be None and 0 may be entered under both the Offset and Value columns.
```

Comment:
Throughout I revised and mostly shortened these mandatory data discussions - they're useful as a reference, but with the input data validation checks they don't need to be extremely detailed.